qa/zcash/full_test_suite.py

```diff
@@ -187,6 +191,10 @@ def rust_test():
     'secp256k1': ['make', '-C', repofile('src/secp256k1'), 'check'],
     'univalue': ['make', '-C', repofile('src/univalue'), 'check'],
     'rpc': [repofile('qa/pull-tester/rpc-tests.py')],
+    'rpc_group0': [repofile('qa/pull-tester/rpc-tests.py'), '-m4', '-r0'],
+    'rpc_group1': [repofile('qa/pull-tester/rpc-tests.py'), '-m4', '-r1'],
+    'rpc_group2': [repofile('qa/pull-tester/rpc-tests.py'), '-m4', '-r2'],
+    'rpc_group3': [repofile('qa/pull-tester/rpc-tests.py'), '-m4', '-r3'],
```

Comment:
Also now understand the ordering @nuttycom mentioned since I didn't even really need to call them in the first place. Ty for reviews, makes sense now with a bit more sleep than last week.